components/changelog/ChangelogTrigger.js

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { gql, useQuery } from '@apollo/client';
+import { graphql, withApollo } from '@apollo/client/react/hoc';
+import themeGet from '@styled-system/theme-get';
+import { cloneDeep } from 'lodash';
+import styled from 'styled-components';
+
+import { API_V2_CONTEXT, gqlV2 } from '../../lib/graphql/helpers';
+import { parseToBoolean } from '../../lib/utils';
+
+import { Flex } from '../Grid';
+import { withNewsAndUpdates } from '../NewsAndUpdatesProvider';
+import { Dropdown } from '../StyledDropdown';
+import StyledRoundButton from '../StyledRoundButton';
+
+import ChangelogNotificationDropdown from './ChangelogNotificationDropdown';
+
+const CHANGE_LOG_UPDATES_ENABLED = parseToBoolean(process.env.CHANGE_LOG_UPDATES_ENABLED);
+
+const FlameIcon = styled(StyledRoundButton)`
+  border-radius: 50%;
+  height: 30px;
+  width: 30px;
+  margin-left: 2px;
+
+  &,
+  &:active {
+    background: ${props => themeGet(props.backgroundColor)};
+    background-image: ${props => `url(${props.url})`};
+    background-repeat: no-repeat;
+    background-position: center center;
+  }
+
+  &:active {
+    background-color: transparent;
+  }
+`;
+
+const ChangelogTrigger = props => {
+  const { setShowNewsAndUpdates, setChangelogViewDate } = props;
+  const { data } = useQuery(loggedInUserQuery, { fetchPolicy: 'cache-only' });
+  const LoggedInUser = data?.LoggedInUser;
+  const hasSeenNewUpdates = LoggedInUser?.hasSeenLatestChangelogEntry;
+
+  const handleShowNewUpdates = () => {
+    setShowNewsAndUpdates(true);
+    setChangelogViewDate({
+      variables: { changelogViewDate: new Date() },
+      optimisticResponse: {
+        setChangelogViewDate: {
+          __typename: 'Individual',
+          hasSeenLatestChangelogEntry: true,
+        },
+      },
```

Comment:
As per https://github.com/opencollective/opencollective-frontend/pull/6510#discussion_r660352208, let's remove this one for now

```suggestion
```

Reply:
@Betree : Thanks. Removed it. 👍🏽